Userland/Libraries/LibCrypto/BigInt/UnsignedBigInteger.cpp

```diff
@@ -318,6 +344,22 @@ void UnsignedBigInteger::set_bit_inplace(size_t bit_index)
     m_cached_hash = 0;
 }
 
+void UnsignedBigInteger::unset_bit_inplace(size_t bit_index)
```

Comment:
the nomenclature is usually set and clear, so `clear_bit_inplace` would be a bit nicer